meetings/2016-12-22/minutes.md

```diff
@@ -0,0 +1,127 @@
+# Node.js Security WG meeting - 2016-12-22
+
+* GH issue: <https://github.com/nodejs/security-wg/issues/6>
+* Meeting audio: <https://youtu.be/fewvQg-DaVk>
+* Previous meeting: <https://github.com/nodejs/security-wg/pull/10>
+
+> Attendees were more than google allowed, so we switched to Uberconference, and
+> lost streaming. We’ll get this working for next meeting.
+
+* Next meeting: TBD
+
+
+## Present
+
+- Rodd Vagg
+- Sam Roberts
+- Devon Rifkin
+- Bryan English
+- Matteo Collina
+- Vladimir de Turckheim
+- William Kapke
+- Mikeal Rogers
+- Johan Bergström
+- James Snell
+- Colin Ihrig
+- Nikita
+- Deian Stefan
+- Hitesh Kanwathirtha
+- Fraser Brown
+
+
+## Standup
+
+- Those present did a quick introduction of themselves.
+
+
+## Agenda
+
+- Introductions
+- What to do with the Node Security Project donation?
+- What is the working group? A TLP, CWG, TWG? See #11
+
+
+## Discussion
+
+- nsp project is getting donated (not signed yet)
+- Some discussion of a set of problems
+- Would like a streamlined way to report vulnerabilities
+- How hasn’t been worked out yet
+- This group is not intended to replace the group that triages core
```

Comment:
I think @mikeal also noted that there needs to be a clear distinction between this group and the core vulnerabilities group, and any infringements should be worked out